<?xml version="1.0"?>
<launch>

  <node pkg="fyp_rl" type="evaluate_model.py" name="evaluate_model" respawn="false" required="true" output="screen"/>
  <rosparam file="$(find fyp_rl)/config/robot_init.yaml" command="load" ns="/train_rl"/>

  <node pkg="fyp_rl" type="bullet_tf_broadcaster" name="bullet_tf_broadcaster" required="true" output="screen"/>

</launch>
